app/src/main/java/fr/groggy/racecontrol/tv/utils/locale/LocaleManager.kt

```diff
@@ -0,0 +1,38 @@
+package fr.groggy.racecontrol.tv.utils.locale
+
+import android.content.res.Configuration
+import android.os.Build
+import android.util.Log
+import javax.inject.Inject
+import javax.inject.Singleton
+
+
+@Singleton
+class LocaleManager @Inject constructor() {
+
+    companion object {
+        private val TAG = LocaleManager::class.simpleName
+    }
+
+    // Currently support by F1 API: ENG, DEU, FRA, NLD, SPA, PRT
+    var currentLocale: String = "ENG"
+        set(iso3Language) {
+            Log.d(TAG, iso3Language)
+            field = when (iso3Language) {
+                "deu" -> "DEU"
+                "fra" -> "FRA"
+                "nld" -> "NLD"
+                "spa" -> "SPA"
+                "por" -> "PRT"
```

Comment:
I think this is `POR`, if I try any of F1 apis with `PRT` it just returns an error.

If that is the case, then this can just be an uppercase of the `iso3Language`

Reply:
Also, then we don't really need the `LocaleManager` class, it can just become a property at `F1TVClient`